docs/getting-started.md

```diff
@@ -181,18 +181,40 @@ token.setAddress('owner', event.params.to)
 token.setU256('amount', event.params.tokens)
 ```
 
-There is also a global `Store` class which has a `set` method for setting the value(s) of a particular entity's attribute(s) in the store.
+There is also a global `Store` class which has a `set` and `get` methods for setting ang getting the value(s) of a particular entity's attribute(s) in the store.
```

Comment:
Grammar: "a set and get methods" -> "set and get methods"
Typo: "ang getting" -> "and getting"